app/assets/stylesheets/components/_alerts.scss

```diff
@@ -0,0 +1,34 @@
+.alert {
+  border-radius: 10px;
+  color: $black;
+  margin-bottom: 10px;
+  padding: 15px;
+}
+
+.alert-success {
```

Comment:
@wontoan There is a slight issue with code-reuse here.  The way you've written it, if I wanted to display a success alert, I'd have to do something like this:  `<div class='alert alert-success'>`.  That's not a horrible thing but it would be more readable to have either `class='alert success'` or `class='alert-success'`.  

For the first, you can use Sass nesting.  This is one of the most powerful parts of using a pre-processor.  For example:

```
.alert {
  border-radius: 10px;
  border-radius: 10px;
  color: $black;
  margin-bottom: 10px;
  padding: 15px;
  
  &.success {
    border: 1px solid $green;
  }
```

If you want to do the latter, you can use Sass's `@extends` feature.  For example:
```
.alert {
  // Your styles here
}

.alert-success {
  @extend .alert;
  border: 1px solid $green;
}
```
Read more here:  http://sass-lang.com/documentation/file.SASS_REFERENCE.html#extend